sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -426,6 +422,12 @@ def test_expand_kafka_write(self):
   def test_sql(self):
     raise unittest.SkipTest("BEAM-7252")
 
+  def test_pack_combiners(self):
+    raise unittest.SkipTest(
```

Comment:
Lets add an optional flag to test_pack_combiners of whether to verify this happened via counters. We can override this here to pass the non-default value of the flag so we can still run the test (and ensure the outputs are correct) despite the issue with using counters to verify the packing.

Reply:
Done.